doc/rest.md

```diff
@@ -527,6 +535,7 @@ JSON object with following structure:
 - **url**: String. URL pointing to this endpoint.
 - **title**: String. Taken from `title` attribute of JSON root object
 - **description**: String. Taken from `abstract` attribute of JSON root object.
+- **updated_at**: Datetime. Date and time of last POST/PATCH of the publication. In format: 'YYYY-MM-DDTHH24:MI:SS.USZ', always in UTC.
```

Comment:
"String. Date and time of last POST/PATCH of the publication. Format is [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601), more specifically `YYYY-MM-DDThh:mm:ss.sss±hh:mm`, always in UTC. Sample value: `"2021-03-18T09:29:53.769233+00:00"`